var/spack/repos/builtin/packages/rocm-rocr-runtime/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class RocmRocrRuntime(CMakePackage):
+    """This repository includes the user-mode API interfaces and
+    libraries necessary for host applications to launch compute
+    kernels to available HSA ROCm kernel agents. Reference source code
+    for the core runtime is also available."""
+
+    homepage = "https://github.com/RadeonOpenCompute/ROCR-Runtime"
+    url = "https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-3.5.0.tar.gz"
+
+    root_cmakelists_dir = "src"
+
+    version('3.5.0', '52c12eec3e3404c0749c70f156229786ee0c3e6d3c979aed9bbaea500fa1f3b8')
+    depends_on('rocm-roct-thunk-interface')
```

Comment:
@aweits Likely also needs a `libelf` dependency here.  

Seeing these types of errors:
```
==> Error: Failed to install rocm-rocr-runtime due to ChildError: ProcessError: Command exited with status 1:
    'cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7.4.0/rocm-rocr-runtime-3.5.0-sas4qfsi3d4qs5uiafhwx44muxiq5jd6' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INS
TALL_RPATH_USE_LINK_PATH:BOOL=OFF' '-DCMAKE_INSTALL_RPATH:STRING=/root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7.4.0/rocm-rocr-runtime-3.5.0-sas4qfsi3d4qs5uiafhwx44muxiq5jd6/lib;/root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7.4.0/rocm-rocr-runtime-3.5.0-sas4q
fsi3d4qs5uiafhwx44muxiq5jd6/lib64;/root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7.4.0/rocm-roct-thunk-interface-3.5.0-unyl6gjbinjcdmwoz4jqhv7472yevfiq/lib' '-DCMAKE_PREFIX_PATH:STRING=/root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7.4.0/cmake-3.17.1-2dczyfsidp
vwsmcr5cnoqhepagk4lqfu;/root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7.4.0/rocm-roct-thunk-interface-3.5.0-unyl6gjbinjcdmwoz4jqhv7472yevfiq' '-DCMAKE_BUILD_WITH_INSTALL_RPATH=1' '/tmp/root/spack-stage/spack-stage-rocm-rocr-runtime-3.5.0-sas4qfsi3d4qs5uiafhwx44muxiq
5jd6/spack-src/src'
1 error found in build log:
     11    -- Check for working CXX compiler: /root/spack/lib/spack/env/gcc/g++
     12    -- Check for working CXX compiler: /root/spack/lib/spack/env/gcc/g++
            - works
     13    -- Detecting CXX compiler ABI info
     14    -- Detecting CXX compiler ABI info - done
     15    -- Detecting CXX compile features
     16    -- Detecting CXX compile features - done
  >> 17    CMake Error at /root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7
           .4.0/cmake-3.17.1-2dczyfsidpvwsmcr5cnoqhepagk4lqfu/share/cmake-3.17/
           Modules/FindPackageHandleStandardArgs.cmake:164 (message):
     18      Could NOT find LibElf (missing: LIBELF_LIBRARIES LIBELF_INCLUDE_DI
           RS)
     19    Call Stack (most recent call first):
     20      /root/spack/opt/spack/linux-ubuntu18.04-nehalem/gcc-7.4.0/cmake-3.
           17.1-2dczyfsidpvwsmcr5cnoqhepagk4lqfu/share/cmake-3.17/Modules/FindP
           ackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
     21      cmake_modules/FindLibElf.cmake:49 (FIND_PACKAGE_HANDLE_STANDARD_AR
           GS)
     22      CMakeLists.txt:72 (find_package)
```

Reply:
@aweits ,  you can use the packages that are already built and tested with rocm-3.5.1 . it is available here at https://github.com/RadeonOpenCompute/rocm-spack/tree/develop/packages . We are looking to raise a PR for these soon

Reply:
I'll be removing the ROCm components from the PR, and we'll go with the packages that the ROCm team has opened PR's for.